EIPS/eip-4573.md

```diff
@@ -0,0 +1,146 @@
+---
+eip: 4573
+title: Entry Points and Procedures for EVM Code Sections
+description: This proposal introduces EVM procedures as entry points to EOF code sections.
+status: Draft
+type: Standards Track 
```

Comment:
```suggestion
type: Standards Track
```
This should satisfy the bot.  There is an issue in the backlog to improve the error messaging around this: https://github.com/lightclient/eipv/issues/20